spec/system/admin/bulk_url_upload_spec.rb

```diff
@@ -37,94 +78,42 @@
   describe 'bulk uploading a file of URLs' do
     include_context 'log in super admin'
 
-    it 'sends us back to the bulk upload page' do
-      bulk_upload
-      expect(page).to have_text('Bulk Search.gov URL Upload')
-    end
+    let(:upload_filename) { 'good_url_file.txt' }
 
-    it 'shows a confirmation message' do
-      bulk_upload
-      expect(page).to have_text(
-        <<~CONFIRMATION_MESSAGE
-          Successfully uploaded #{url_filename} for processing.
-          The results will be emailed to you.
-        CONFIRMATION_MESSAGE
-      )
-    end
+    it_behaves_like 'a successful bulk upload'
+  end
 
-    it 'creates the URLs' do
-      bulk_upload
-      urls.each do |url|
-        expect(SearchgovUrl.find_by(url: url)).not_to be_blank
-      end
-    end
+  describe 'bulk uploading a UTF-8 file of URLs' do
```

Comment:
See proposed updated spec above. Note the use of a more explicit `context` rather than a `describe`, and the omission of the duplicative shared examples.

Reply:
Changed.

Reply:
Changed.